tests/unit/optimizer_test.py

```diff
@@ -19,170 +19,175 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-"""Tests nab.optimizer for finding the local/global maxima of several 
-functions"""
-
-import math
+import datetime
 import unittest
 
-from nab.optimizer import twiddle
+import pandas
```

Comment:
pandas is a third party library so it isn't in the same section as the built in modules (above) or the NAB imports (below)